libqtile/notify.py

```diff
@@ -22,63 +23,55 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-"""
-    If dbus is available, this module implements a
-    org.freedesktop.Notifications service.
-"""
-from libqtile.log_utils import logger
+# type: ignore
+
+from dbus_next.aio import MessageBus  # noqa: I001
```

Comment:
Instead of noqaing these, would it be possible to run `tox -e format`? It should automatically re-order the imports to make flake8 happy. Otherwise, they just need to be in alphabetical order I think.

Reply:
I think these might have been ok. Neither tox or isort seemed to change them. I've removed the noqa.